### ISSUE TITLE
Graphql error with update to 1.5.4

### ISSUE THREAD

**Issue by janeschindler** [verbb/formie#704]
**Description**
Craft 3.7.28 and formie 1.5.4 cause graphql errors
**Steps to reproduce**
Install craft 3.7.28 and formie 1.5.4.
make a graphql query like
`      form {
        handle
        settings {
          errorMessageHtml
          submitActionMessageHtml
        }
        pages {
          rows {
            fields {
              id
              name
              placeholder
              required
              type
              handle
              displayName
              ...on Field_Agree {
                descriptionHtml
              }
            }
          }
          settings {
            submitButtonLabel
          }
        }
      }
`
See error 

`{
  "errors": [
    {
      "debugMessage": "Argument 1 passed to verbb\\formie\\gql\\interfaces\\PageInterface::verbb\\formie\\gql\\interfaces\\{closure}() must be an instance of verbb\\formie\\models\\FieldLayoutPage, instance of craft\\models\\FieldLayoutTab given, called in /Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Type/Definition/InterfaceType.php on line 115",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "file": "/Users/janeschindler/Documents/server/ggcms/vendor/verbb/formie/src/gql/interfaces/PageInterface.php",
      "line": 41,
      "trace": [
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Type/Definition/InterfaceType.php",
          "line": 115,
          "call": "verbb\\formie\\gql\\interfaces\\PageInterface::verbb\\formie\\gql\\interfaces\\{closure}(instance of craft\\models\\FieldLayoutTab, array(2), instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 941,
          "call": "GraphQL\\Type\\Definition\\InterfaceType::resolveType(instance of craft\\models\\FieldLayoutTab, array(2), instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 790,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: PageInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(7), instance of craft\\models\\FieldLayoutTab)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: PageInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(7), instance of craft\\models\\FieldLayoutTab)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 888,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: PageInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(7), instance of craft\\models\\FieldLayoutTab)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 762,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeListValue(GraphQLType: PageInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(6), array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: PageInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(6), array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: PageInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(6), array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1196,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Form, instance of verbb\\formie\\elements\\Form, instance of ArrayObject(1), array(6))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1146,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Form, instance of verbb\\formie\\elements\\Form, array(5), instance of ArrayObject(3))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1107,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: Form, instance of ArrayObject(1), array(5), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 975,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: Form, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(5), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 790,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(5), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(5), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 888,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(5), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 762,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeListValue(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(4), array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(4), array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(4), array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1196,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: aboutUsContent_form_BlockType, instance of benf\\neo\\elements\\Block, instance of ArrayObject(1), array(4))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1146,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: aboutUsContent_form_BlockType, instance of benf\\neo\\elements\\Block, array(3), instance of ArrayObject(3))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1107,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: aboutUsContent_form_BlockType, instance of ArrayObject(1), array(3), instance of benf\\neo\\elements\\Block)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 975,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: aboutUsContent_form_BlockType, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of benf\\neo\\elements\\Block)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 790,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: aboutUsContent_NeoField, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of benf\\neo\\elements\\Block)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: aboutUsContent_NeoField, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of benf\\neo\\elements\\Block)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 888,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: aboutUsContent_NeoField, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of benf\\neo\\elements\\Block)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 762,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeListValue(GraphQLType: aboutUsContent_NeoField, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(9))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: aboutUsContent_NeoField, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(9))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: aboutUsContent_NeoField, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(9))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1196,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: aboutUs_aboutUs_Entry, instance of craft\\elements\\Entry, instance of ArrayObject(1), array(2))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1146,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: aboutUs_aboutUs_Entry, instance of craft\\elements\\Entry, array(1), instance of ArrayObject(3))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1107,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: aboutUs_aboutUs_Entry, instance of ArrayObject(1), array(1), instance of craft\\elements\\Entry)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 975,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: aboutUs_aboutUs_Entry, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of craft\\elements\\Entry)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 790,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of craft\\elements\\Entry)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of craft\\elements\\Entry)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of craft\\elements\\Entry)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1196,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Query, null, instance of ArrayObject(1), array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 264,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Query, null, array(0), instance of ArrayObject(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 215,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeOperation(instance of GraphQL\\Language\\AST\\OperationDefinitionNode, null)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/Executor/Executor.php",
          "line": 156,
          "call": "GraphQL\\Executor\\ReferenceExecutor::doExecute()"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 162,
          "call": "GraphQL\\Executor\\Executor::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, instance of GraphQL\\Language\\AST\\DocumentNode, null, array(2), null, 'aboutUsQuery', null)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 94,
          "call": "GraphQL\\GraphQL::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, 'query aboutUsQuery {\n  aboutUs: entry(slug: \"about-us\") {\n    id\n    slug\n    ... on aboutUs_aboutUs_Entry {\n      aboutUsContent {\n                 \n        ...on aboutUsContent_form_BlockType {\n          heading: text\n          desription: richText\n          form {\n            handle\n            settings {\n              errorMessageHtml\n              submitActionMessageHtml\n        \t\t}\n            pages {\n              rows {\n                fields {\n                  id\n                  name\n                  placeholder\n                  required\n                  type\n                  handle\n                  displayName\n                  ...on Field_Agree {\n                    descriptionHtml\n                  }\n                }\n              }\n              settings {\n                submitButtonLabel\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n', null, array(2), null, 'aboutUsQuery', null, array(26))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/craftcms/cms/src/services/Gql.php",
          "line": 544,
          "call": "GraphQL\\GraphQL::executeQuery(instance of GraphQL\\Type\\Schema, 'query aboutUsQuery {\n  aboutUs: entry(slug: \"about-us\") {\n    id\n    slug\n    ... on aboutUs_aboutUs_Entry {\n      aboutUsContent {\n                 \n        ...on aboutUsContent_form_BlockType {\n          heading: text\n          desription: richText\n          form {\n            handle\n            settings {\n              errorMessageHtml\n              submitActionMessageHtml\n        \t\t}\n            pages {\n              rows {\n                fields {\n                  id\n                  name\n                  placeholder\n                  required\n                  type\n                  handle\n                  displayName\n                  ...on Field_Agree {\n                    descriptionHtml\n                  }\n                }\n              }\n              settings {\n                submitButtonLabel\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n', null, array(2), null, 'aboutUsQuery', null, array(26))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/craftcms/cms/src/controllers/GraphqlController.php",
          "line": 172,
          "call": "craft\\services\\Gql::executeQuery(instance of craft\\models\\GqlSchema, 'query aboutUsQuery {\n  aboutUs: entry(slug: \"about-us\") {\n    id\n    slug\n    ... on aboutUs_aboutUs_Entry {\n      aboutUsContent {\n                 \n        ...on aboutUsContent_form_BlockType {\n          heading: text\n          desription: richText\n          form {\n            handle\n            settings {\n              errorMessageHtml\n              submitActionMessageHtml\n        \t\t}\n            pages {\n              rows {\n                fields {\n                  id\n                  name\n                  placeholder\n                  required\n                  type\n                  handle\n                  displayName\n                  ...on Field_Agree {\n                    descriptionHtml\n                  }\n                }\n              }\n              settings {\n                submitButtonLabel\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}\n', null, 'aboutUsQuery', true)"
        },
        {
          "call": "craft\\controllers\\GraphqlController::actionApi()"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/yiisoft/yii2/base/InlineAction.php",
          "line": 57,
          "function": "call_user_func_array(array(2), array(0))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/yiisoft/yii2/base/Controller.php",
          "line": 178,
          "call": "yii\\base\\InlineAction::runWithParams(array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/yiisoft/yii2/base/Module.php",
          "line": 552,
          "call": "yii\\base\\Controller::runAction('api', array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/craftcms/cms/src/web/Application.php",
          "line": 287,
          "call": "yii\\base\\Module::runAction('graphql/api', array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/craftcms/cms/src/web/Application.php",
          "line": 596,
          "call": "craft\\web\\Application::runAction('graphql/api', array(1))"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/craftcms/cms/src/web/Application.php",
          "line": 266,
          "call": "craft\\web\\Application::_processActionRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/vendor/yiisoft/yii2/base/Application.php",
          "line": 384,
          "call": "craft\\web\\Application::handleRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/Users/janeschindler/Documents/server/ggcms/web/index.php",
          "line": 26,
          "call": "yii\\base\\Application::run()"
        }
      ]
    }
  ],
  "data": {
    "aboutUs": {
      "id": "11381",
      "slug": "about-us",
      "aboutUsContent": [
        {},
        {},
        {},
        {},
        {},
        {},
        {},
        {},
        {
          "heading": "Unterstützen Sie uns!",
          "desription": "<p>Das Thema Geschlecht betrifft uns alle – oft auf einer ganz persönlichen Ebene. Die Debatte muss deshalb respektvoll und gemeinsam geführt werden. Mit #geschlechtergerechter steigen wir in die Stiefel anderer, erweitern unsere Horizonte, halten zusammen und bringen so die Chancengleichheit in der Schweiz voran. Klingt spannend? Dann werden auch Sie Teil davon!</p>",
          "form": [
            {
              "handle": "parterWerden",
              "settings": {
                "errorMessageHtml": "Couldn’t save the submission due to some errors.",
                "submitActionMessageHtml": "Danke für Ihre Eingabe!"
              },
              "pages": [
                null
              ]
            }
          ]
        }
      ]
    }
  }
}`
### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.4
- Craft version: 3.7.28
- Multi-site: No

**Additional context**
Probably related to the other two newest issues.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.5](https://github.com/verbb/formie/releases/tag/1.5.5)
